lab_6_pipeline/pipeline.py

```diff
@@ -144,16 +217,49 @@ def __init__(self, corpus_manager: CorpusManager):
         """
         Initializes MorphologicalAnalysisPipeline
         """
+        self._corpus = corpus_manager
 
     def _process(self, text: str) -> List[ConlluSentence]:
         """
         Returns the text representation as the list of ConlluSentence
         """
+        result_list = []
+        sent_list = split_by_sentence(text)
+        for position, element in enumerate(sent_list):
+            conllu_tokens = []
+            words = ConlluToken(element).get_cleaned().split()
+
+            for one_word in words:
+                conllu_tokens.append(ConlluToken(one_word))
+
+            result_list.append(ConlluSentence(position=position,
+                                              text=element,
+                                              tokens=conllu_tokens))
+        return result_list
 
     def run(self) -> None:
         """
         Performs basic preprocessing and writes processed text to files
         """
+        art_dict = self._corpus.get_articles()
```

Comment:
general recommendations: 

1. use full names: `art_dict` -> `article_dict`
2. do not use types names in variable names